libsgfdata/__init__.py

```diff
@@ -241,7 +241,7 @@ def projection(self):
     def positions(self):
         import geopandas as gpd
 
-        assert self.main
+        assert len(self.main)>0,'self.main DataFrame is either missing or is empty'
```

Comment:
Need a check that it isn't None too...

Reply:
Good call. I've made a new commit adding that check